app/assets/javascripts/edit_review.js

```diff
@@ -0,0 +1,60 @@
+$(".edit_button").click(function (event) {
+  event.preventDefault();
+  var href = this.href.split("/");
+  var review_id = href[href.length - 2];
+  if (this.text == "Edit Review") {
+    var rating_string = $('#review_rating' + review_id).text().split(" ");
+    var description = $('#review_description' + review_id).text();
+    var rating = rating_string[rating_string.length - 1][0];
+
+    $('.review_item' + review_id).remove();
+    $('.column#rating' + review_id).append(select_box(rating, review_id));
+    $('.column#description' + review_id).append(text_box(description, review_id));
+    this.text = "Resubmit Review";
+  } else {
+    var new_rating = $("#edit_rating" + review_id).val();
```

Comment:
Identifier 'new_rating' is not in camel case.<br>Identifier 'review_id' is not in camel case.